### DISCUSSION TITLE
No adding shortcut when apps get updated

### DISCUSSION THREAD

**Discussion by DrNopt** [Devolutions/UniGetUI#2355] [General]
Hello,
I would like to request an option to "don't add shortcuts do desktop" when apps get updated. 
It's annoying and we should be able to deny it.
Thank you

**Comment by marticliment** [comment]
Hello,

Software installations are handled by thr package managers, and not by WingetUI/UniGetUI itself. Therefore, it is impossible to fix this from WingetUI.

**Comment by DrNopt** [comment]
PatchMyPC have the option to remove shortcuts. So, it shouldn't be impossible.

**Reply by marticliment** [reply]
It is not that it would be _physically impossible_, but WingetUI would need to be rewritten entirely, since it does not manage uninstalls directly.
